third_party/move/move-compiler/src/to_bytecode/translate.rs

```diff
@@ -618,6 +618,7 @@ fn visibility(v: Visibility) -> IR::FunctionVisibility {
         Visibility::Public(_) => IR::FunctionVisibility::Public,
         Visibility::Friend(_) => IR::FunctionVisibility::Friend,
         Visibility::Internal => IR::FunctionVisibility::Internal,
+        _ => unreachable!(),
```

Comment:
Please panic with an error message 'unexpected visibility'. `unreachable!` should only be used if its clearly never failing, e.g. if you made a check just before in the local code (`if x.is_some() { let Some(y) = x else { unreachable!() }}`)

Reply:
done